app/controllers/application_controller.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  before_action :set_item_search
+
+  def set_item_search
+    @item_search = ItemSearch.new
```

Comment:
I'm eager for us to come up with a clever way to generalize this search function for all models.